src/tests/components/BoardOptionsMenu/ColourOptions/__snapshots__/ColourOptions.test.tsx.snap

```diff
@@ -0,0 +1,180 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`ColourOptions should render successfully 1`] = `
+<DocumentFragment>
```

Comment:
These snap files are auto generated by jest and should be committed. Though they can be ignored for the purposes of review